### DISCUSSION TITLE
OpenAPI Callback

### DISCUSSION THREAD

**Discussion by yeaharh** [microsoft/typespec#2693] [Q&A]
Does Typespec support OpenAPI callbacks? Are there any examples to look at?

**Comment by markcowl** [comment]
Unfortunately, not yet.  As part of enhanced support for streaming APIs, events with callbacks could be modeled in this way.  We are currently tracking the design of this in https://github.com/microsoft/typespec/issues/154  .  Please feel free to add in your comments.
